lib/brainstem/dsl/base_block.rb

```diff
@@ -22,8 +22,23 @@ def descend(klass, new_config = configuration, new_options = {}, &block)
 
         def parse_args(args)
           options = args.last.is_a?(Hash) ? args.pop : {}
-          description = args.shift
-          [description, options]
+          maybe_description = args.shift
+
+          if maybe_description.is_a?(Hash)
```

Comment:
Avoiding `is_a?` could be nice — `kind_of?` would be a nominal improvement, but if we could depend on how these objects quack it would be better still.